Classes/Hooks/FileDumpHook.php

```diff
@@ -199,9 +199,15 @@ protected function dumpFileContents($file, $asDownload, $resumableDownload)
             $file->getStorage()->dumpFileContents($file, $asDownload);
             exit;
         }
+        $download_name = $file->getProperty('download_name') ? $file->getProperty('download_name') : $file->getName();
```

Comment:
Can you change this to:

`$downloadName = $file->getProperty('download_name') ?: $file->getName();`

Use lowerCamelCase for the variable name (also for `$fileParts`) and shorten the comparison by using `?:`.